### ISSUE TITLE
span columns throught multi rows

### ISSUE THREAD

**Issue by sulram** [theprotein/sharps#133]
It would be nice multi rows implementation, span columns throught multi rows.
How do you recommend doing this?

**Comment by awinogradov** [comment]
What do you when say multi rows? You want to have columns through rows and simple columns in the one place?

Like this?
```
-------------------
   |   |     |    |
   -----     ------
   |   |     |    |
-------------------
```

**Comment by sulram** [comment]
what I'm calling multi rows is wrapping columns and creating automatic rows, like the code bellow:

    <div class="row">
        <div class="row__col_sw_6 row__col_mw_4"></div>
        <div class="row__col_sw_6 row__col_mw_4"></div>
        <div class="row__col_sw_6 row__col_mw_4"></div>
        <div class="row__col_sw_6 row__col_mw_4"></div>
        <div class="row__col_sw_6 row__col_mw_4"></div>
        <div class="row__col_sw_6 row__col_mw_4"></div>
    </div>


The idea is to create a grid of posts, 3 rows and 2 columns on mobile and 2 rows and 3 columns on larger screens.

**Comment by awinogradov** [comment]
Yeah, I understand. It looks like an endless grid. But I don't have an idea how to do this in current code state. I think it needs big code changes, but why not ;)
